src/resources/views/crud/inc/form_fields_script.blade.php

```diff
@@ -0,0 +1,183 @@
+            if (input.length !== 0) {
+                return input;
+            }
+
+            // otherwise, try to find the input using other selectors
+            if (this.$input.length === 0) {
+                // try searching for the field with the corresponding bp-field-name
+                input = this.wrapper.find('input[bp-field-name="'+this.name+'"], textarea[bp-field-name="'+this.name+'"], select[bp-field-name="'+this.name+'"], select[bp-field-name="'+this.name+'[]"]').first();
+
+                // if not input found yet, just get the first input in that wrapper
+                if (input.length === 0) {
+                    input = this.wrapper.find('input, textarea, select').first();
+                }
+
+                return input;
+            }
+
+            return this.$input;
+
+        }
+
+        get value() {
+            return this.$input.val();
+        }
+
+        get inputWrapper() {
+            let wrapper = this.$input.closest('[bp-field-wrapper]');
+            if (wrapper.length === 0) {
+                wrapper = $(`[bp-field-name="${this.name}"][bp-field-wrapper]`).first();
+            }
+            return wrapper;
+        }
+
+        onChange(closure) {
+            const bindedClosure = closure.bind(this);
+            const fieldChanged = (event, values) => bindedClosure(this, event, values);
+
+            if(this.isSubfield) {
+                window.crud.subfieldsCallbacks = window.crud.subfieldsCallbacks ?? [];
+                window.crud.subfieldsCallbacks[this.subfieldHolder] = window.crud.subfieldsCallbacks[this.subfieldHolder] ?? [];
+                if(!window.crud.subfieldsCallbacks[this.subfieldHolder].some( callbacks => callbacks['fieldName'] === this.name )) {
+                    window.crud.subfieldsCallbacks[this.subfieldHolder].push({closure: closure, field: this});
+                }
+                return this;
+            }
+
+            if(['INPUT', 'TEXTAREA'].includes(this.input?.nodeName)) {
+                this.input?.addEventListener('input', fieldChanged, false);
+            }
+            this.$input.change(fieldChanged);
+
+            return this;
+        }
+
+        change() {
+            if(this.isSubfield) {
+                if(typeof window.crud.subfieldsCallbacks[this.subfieldHolder].length !== 'undefined') {
+                    window.crud.subfieldsCallbacks[this.subfieldHolder].forEach(item => {
+                        item.triggerChange = true;
+                    });
+                }
+                return this;
+            }
+
+            this.$input.trigger(`change`);
+        }
+
+        show(value = true) {
+            this.wrapper.toggleClass('d-none', !value);
+            this.$input.trigger(`CrudField:${value ? 'show' : 'hide'}`);
+            return this;
+        }
+
+        hide(value = true) {
+            return this.show(!value);
+        }
```

Comment:
Sure, It's because the name of that field is not added in the `fields.js` field list.

I have it here, will submit the PR in a few moments.

Cheers